src/CloudKit/CKCompat.cs

```diff
@@ -29,14 +29,6 @@ public CKNotificationID ()
 #endif
 
 #if XAMCORE_2_0 || !MONOMAC
-	public partial class CKFetchNotificationChangesOperation {
-
-		// `init` does not work on watchOS but we can keep compatibility with a different init
-		public CKFetchNotificationChangesOperation () : this ((CKServerChangeToken) null)
```

Comment:
I just did this on my 3.0 device:

```
CKFetchNotificationChangesOperation *obj = [[CKFetchNotificationChangesOperation alloc] init];
NSLog (@"obj: %@", obj);
```

and it worked fine, so I guess this is OK.